fej-snikduj/URLColors
# Manifest-v3 upgrade plus wildcard support

Should migrate URLColors over to manifest V3s event based architecture while also introducing wildcard support, the most requested feature.

Testing instructions:
1) pull latest code
2) remove existing extension
3) go to manage extensions
4) load unpacked extension, and select URLColors folder
5) pin it to the bar for easy access

Things to test:
1) dismissable wildcard banner.  Should only ever appear once for you.  never again after dismissed
2) test wildcard functionality.  For example, chat*.com should work for chat.openai.com
3) test snooze functionality.  Now supports non integers for easier testing (input 0.1)
4) general testing of settings to make sure we aren't introducing a regression.  6000 users so gotta make sure not to blow things up



<img width="882" alt="image" src="https://github.com/fej-snikduj/URLColors/assets/20237437/aee412d8-ecc8-4225-9826-aa6c317aa848">
